src/materialized/bin/materialized.rs

```diff
@@ -112,6 +113,12 @@ fn run() -> Result<(), failure::Error> {
         "where materialized will write logs (default <data directory>/materialized.log)",
         "PATH",
     );
+    opts.optopt(
+        "",
+        "bind-to",
```

Comment:
Can we call this `listen-addr`? That's what I typically see used as the name and it parallels the PostgreSQL connection parameter: https://www.postgresql.org/docs/current/runtime-config-connection.html